src/features/courses/course-controller.ts

```diff
@@ -1220,6 +1220,16 @@ class CourseController {
             if (studentGrade.courseWWTopicQuestionId !== options.questionId) {
                 throw new NotFoundError('The workbook you have requested does not belong to the question provided');
             }
+            // make sure to grab the right path & seed if this is an assessment workbook!
+            if (_.isNil(workbook.studentGradeInstanceId)) {
+                problemSeed = studentGrade.randomSeed;
+                numIncorrect = studentGrade.numAttempts;
+            } else {
+                gradeInstance = await workbook.getStudentGradeInstance();
```

Comment:
Should probably include:
```Typescript
{
where: {
active: true
}
}
```

Reply:
This should never be the case, unless somehow we decide it can be in the future, so it's fine for now